spec/feature/post_peep_spec.rb

```diff
@@ -0,0 +1,10 @@
+feature 'the user can' do
+  scenario 'post a peep' do
+    visit'/peep/new'
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.